sdk/documentintelligence/ai-document-intelligence-rest/review/ai-document-intelligence.api.md

```diff
@@ -2013,37 +2159,63 @@ export interface ListOperationsHeaders {
 export type ListOperationsParameters = ListOperationsHeaderParam & RequestParameters;
 
 // @public
-export type OperationDetailsOutput = OperationDetailsOutputParent | DocumentModelBuildOperationDetailsOutput | DocumentModelComposeOperationDetailsOutput | DocumentModelCopyToOperationDetailsOutput | DocumentClassifierCopyToOperationDetailsOutput | DocumentClassifierBuildOperationDetailsOutput;
+export interface ModelCopyAuthorization {
+    accessToken: string;
+    expirationDateTime: Date | string;
+    targetModelId: string;
+    targetModelLocation: string;
+    targetResourceId: string;
+    targetResourceRegion: string;
+}
 
 // @public
-export interface OperationDetailsOutputParent {
-    apiVersion?: string;
-    createdDateTime: string;
-    error?: ErrorModelOutput;
-    // (undocumented)
-    kind: OperationKindOutput;
-    lastUpdatedDateTime: string;
-    operationId: string;
-    percentCompleted?: number;
-    resourceLocation: string;
-    status: OperationStatusOutput;
-    tags?: Record<string, string>;
+export interface ModelCopyAuthorizationOutput {
+    accessToken: string;
+    expirationDateTime: string;
+    targetModelId: string;
+    targetModelLocation: string;
+    targetResourceId: string;
+    targetResourceRegion: string;
 }
 
 // @public
 export type OperationKindOutput = string;
 
 // @public
-export type OperationStatusOutput = string;
+export interface PagedAnalyzeBatchOperationOutput {
+    nextLink?: string;
+    value: Array<AnalyzeBatchOperationOutput>;
+}
 
 // @public
-export type PagedDocumentClassifierDetailsOutput = Paged<DocumentClassifierDetailsOutput>;
+export interface PagedAsyncIterableIterator<TElement, TPage = TElement[], TPageSettings = PageSettings> {
+    [Symbol.asyncIterator](): PagedAsyncIterableIterator<TElement, TPage, TPageSettings>;
+    byPage: (settings?: TPageSettings) => AsyncIterableIterator<TPage>;
+    next(): Promise<IteratorResult<TElement>>;
+}
 
 // @public
-export type PagedDocumentModelDetailsOutput = Paged<DocumentModelDetailsOutput>;
+export interface PagedDocumentClassifierDetailsOutput {
+    nextLink?: string;
+    value: Array<DocumentClassifierDetailsOutput>;
+}
 
 // @public
-export type PagedOperationDetailsOutput = Paged<OperationDetailsOutput>;
+export interface PagedDocumentIntelligenceOperationDetailsOutput {
+    nextLink?: string;
+    value: Array<DocumentIntelligenceOperationDetailsOutput>;
+}
+
+// @public
+export interface PagedDocumentModelDetailsOutput {
+    nextLink?: string;
+    value: Array<DocumentModelDetailsOutput>;
+}
+
+// @public
+export interface PageSettings {
+    continuationToken?: string;
+}
 
 // @public
 export function paginate<TResponse extends PathUncheckedResponse>(client: Client, initialResponse: TResponse, options?: PagingOptions<TResponse>): PagedAsyncIterableIterator<PaginateReturn<TResponse>>;
```

Comment:
`PagingOptions` shouldn't be exported to our users, it is meant to be used by our code gen only.